user-service/controller/user-controller.js

```diff
@@ -24,3 +26,20 @@ export async function createUser(req, res) {
     }
 }
 
+export async function signIn(req, res) {
+    try {
+        const { username, password } = req.body;
+        const user = await _FindOneuser(username,password)
+        if (user) {
+            const accessToken = jwt.sign({username:username},ACCESS_TOKENT_SECRET)
+            console.log(accessToken)
+            // res.json({accessToken:accessToken})
+            res.cookie("accesstoken",accessToken,{httpOnly:true})
+            return res.status(201).json({message: `Log in is successful!`,accesstoken:accessToken });

```

Comment:
Response status should be 200, not 201, since in signIn post request no new data entry is being created.

Reply:
Fixed